samples/subsys/video/capture/README.rst

```diff
@@ -7,9 +7,8 @@
 Description
 ***********
 
-This sample application uses the :ref:`Video API <video_api>` to retrieve video frames from the
-video capture device, writes a frame count message to the console, and then
-discards the video frame data.
+This sample application uses the :ref:`Video API <video_api>` to capture frames from
```

Comment:
These changes should be in the commit updating the Video capture sample

Reply:
Yes, thanks.

Reply:
done